Lib/test/test_timeit.py

```diff
@@ -359,11 +359,27 @@ def autorange(self, seconds_per_increment=1/1024, callback=None):
         t = timeit.Timer(stmt=self.fake_stmt, setup=self.fake_setup, timer=timer)
         return t.autorange(callback)
 
+    def autorange_with_argument(self, total_time, seconds_per_increment=1/1024,
+            callback=None):
+        timer = FakeTimer(seconds_per_increment=seconds_per_increment)
+        t = timeit.Timer(stmt=self.fake_stmt, setup=self.fake_setup, timer=timer)
+        return t.autorange(callback, total_time)
+
     def test_autorange(self):
         num_loops, time_taken = self.autorange()
         self.assertEqual(num_loops, 500)
         self.assertEqual(time_taken, 500/1024)
 
+    def test_autorange_with_argument(self):
+        num_loops, time_taken = self.autorange_with_argument(total_time=0.6)
+        self.assertEqual(num_loops, 1000)
+        self.assertEqual(time_taken, 1000/1024)
+
+    def test_autorange_with_argument(self):
```

Comment:
The name of this test seems to be duplicate, so the previous test will be overwritten.
Instead of using `with_argument`, I think it would be better to say what argument you are testing, e.g. `test_autorange_with_total_time`.
You can also merge the two tests, since they are testing the same thing.